src/shared/mod.rs

```diff
@@ -94,11 +100,60 @@ pub fn apply_changes(
     original
 }
 
+pub fn create_ast_package(
+    uri: String,
+    ctx: RequestContext,
+) -> Result<flux::ast::Package, String> {
+    let values =
+        cache::get_package(uri.clone(), ctx.support_multiple_files)?;
```

Comment:
The get_package will require all the files in the package already opened, then they can be cached right? What if a file is not opened, will the parser returns some kind of error?

Reply:
This is why we are going to disable it in vsflux for the time being, we will need to figure out the best way to prime the cache.